package.json

```diff
@@ -16,15 +16,14 @@
   },
   "homepage": "https://github.com/ralphtheninja/pgdown#readme",
   "dependencies": {
-    "abstract-leveldown": "~2.4.1",
```

Comment:
I'll push up another commit that implements batch much cleaner as `_createWriteStream` that should make it more clear.